src/permutat.h

```diff
@@ -56,25 +56,39 @@ static inline UInt DEG_PERM4(Obj perm)   {
   return (SIZE_OBJ(perm) - sizeof(Obj)) / sizeof(UInt4);
 }
 
-static inline UInt4* ADDR_PERM4(Obj perm) {
-  return (UInt4*)(ADDR_OBJ(perm)+1);
+static inline UInt4 * ADDR_PERM4(Obj perm)
+{
```

Comment:
`ADDR_PERM4` etc. are added as static inline functions in one commit, then reformatted in another. Ideally, they'd just be added in a single commit, in their "correct" form.